atomicapp/applogging.py

```diff
@@ -27,15 +27,16 @@
 class customOutputFormatter(logging.Formatter):
     """
     A class that adds 'longerfilename' support to the logging formatter
-    This 'longerfilename' will be filename + parent dir.
+    This 'longerfilename' will be path/to/file.py after the root atomicapp
+    folder.
     """
 
     def format(self, record):
 
         # Add the 'longerfilename' field to the record dict. This is
         # then used by the Formatter in the logging library when
         # formatting the message string.
-        record.longerfilename = '/'.join(record.pathname.split('/')[-2:])
+        record.longerfilename = record.pathname.split(customOutputFormatter.path)[-1].lstrip('/')
```

Comment:
I would do something like this:

```
# The root directory where the atomicapp code lives. Since applogging.py is in atomicapp/applogging.py that should be 1 directory above __file__.
atomicapproot = '/'.join(__file__.split('/')[:-1])
```

And then here I would do:

```
Get the part of the path that is under the root of atomic app
record.longerfilename = record.pathname[len(atomicapproot):]
```

Or something like that